exporter/opentelemetry-exporter-richconsole/tests/test_rich_exporter.py

```diff
@@ -45,3 +48,34 @@ def test_span_exporter(tracer_provider, span_processor, capsys):
     span_processor.force_flush()
     captured = capsys.readouterr()
     assert "V4LuE" in captured.out
+
+
+def walk_tree(root: Tree) -> int:
+    # counts the amount of spans in a tree that contains a span
+    return sum([walk_tree(child) for child in root.children]) + int(
+        "span" in root.label
+    )
+
+
+def test_multiple_traces(tracer_provider):
```

Comment:
Please add a test that checks that the parent-child relationship between spans is correct.

Reply:
Done :)